shared/menubar/index.js

```diff
@@ -139,6 +140,17 @@ class Menubar extends Component<void, Props, void> {
     this._closeMenubar()
   }
 
+  _openShell () {
+    if (process.platform === 'win32') {
+      let shellCmd = 'start "Keybase Shell" "%APPDATA%\\Microsoft\\Windows\\Start Menu\\Programs\\Keybase\\Keybase Shell.lnk"'
```

Comment:
We should make it point to shortcut in app directory, but we need to have installer copy it there first? https://keybase.atlassian.net/browse/DESKTOP-1681